src/pages/ApplyMain/index.js

```diff
@@ -15,14 +15,27 @@ const ApplyMainPage = () => {
   const [innerWidth, setInnerWidth] = useState(window.innerWidth);
   const DESKTOP_WIDTH = 1199;
   const TABLET_WIDTH = 599;
+  const [clikedArray, setClickedArray] = useState([false, false, false]);
```

Comment:
여기 오타있어요..!
```suggestion
  const [clickedArray, setClickedArray] = useState([false, false, false]);
```

요거 `Commit suggestion` 버튼 클릭하시면 바로 수정 가능합니다!

Reply:
세상이.. 좋아졌군요..